src/Orangebeard.Client/Entities/LogLevel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orangebeard.Client.Entities
+{
+    public enum LogLevel
+    {
+        //TODO?~ Determine which ordering to use...
+
+        // This is the ordering in the old .NET Client:
+        trace, debug, info, warn, error, fatal, unknown
```

Comment:
I think in combination with the new UI we can limit this to `error, warn, info, debug`

Reply:
Done.